sample/UseMaven/GNAdGoogleRewardMediationSample/app/build.gradle

```diff
@@ -52,4 +44,6 @@ dependencies {
     // Vungle Optional Google Play Services
     implementation 'com.google.android.gms:play-services-location:17.0.0'
 
+    // TikTok AAR integration
```

Comment:
In the sample app, implementation of Tiktok library is added, but it is not necessary.

Reply:
TikTok was added automatically after I executed this command `./Shells/create-github-projects.sh` from `ssp-android-sdk-samples`

Do I need to remove it?

Reply:
Yes, please remove it.

Reply:
@hironobu-yazaki Thank you. I already removed it on my latest commit